images/linux/scripts/installers/validate-disk-space.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+################################################################################
+##  File:  validate-disk-space.sh
+##  Desc:  Validate free disk space
+################################################################################
+
+availableSpace=$(df / | sed 1d | awk '{ print $4 "\t" }')
+minimumFreeSpaceGB=$(( 15*1024*1024 ))
```

Comment:
Why 15? Have you seen my comment here?
https://github.com/actions/virtual-environments/issues/752#issuecomment-617378137

Reply:
Sorry, I missed your comment.
Added additional 8GB